sensor_msgs/Image[] images
# Descriptor[] descriptors
---
float64[] confidences